README.md

```diff
@@ -16,12 +18,24 @@ cd psammead && npm install
 
 NB, this automatically runs `npm run install:packages` in a postinstall step for you.
 
-### Run Storybook
+## Storybook
 
+### Run Locally
 ```
 npm run storybook
 ```
 
+### Deploying
+
+The Psammead storybook is hosted on github pages at http://bbc-news.github.io/psammead. It is currently deployed via a local script that builds storybook to the `gh-pages` git branch which is used by github pages.
+
+```
+rpm run deploy-storybook
+```
+
+NB, this automatically pushes and deploys, please only run this script on the `latest` branch.
```

Comment:
Isn't "pushes" and "deploys" the same thing?

Reply:
Errm I dont think so, pushing is to a branch, deploy is to an 'enviroment', our 'enviroment' being github pages live

Reply:
Oh ok, could you clarify `automatically pushes to the 'gh-pages' branch, which deploys to the live GitHub pages site'.